src/System.Text.Json/tests/WritableJsonApiTests.JsonNumberUnitTests.cs

```diff
@@ -0,0 +1,473 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Json
+{
+    public static partial class WritableJsonApiTests
```

Comment:
I would rename this to JsonNodeTests instead.
Same with the filename.
```suggestion
    public static partial class JsonNodeTests
```

In this case:
JsonNodeTests.JsonNumber.cs

(UnitTests suffix is redundant, imo)